tests/UnitTests/ApiConnectors/BankTransactionApiConnectorTest.php

```diff
@@ -195,4 +196,33 @@ public function testSendAllReturnsMappedObjectsAllLineTypes()
         $this->assertEquals(Money::EUR(10000), $line->getVatTurnover());
         $this->assertEquals(Money::EUR(10000), $line->getVatBaseTurnover());
     }
+
+    /**
+     * @expectedException Exception
+     * @expectedExceptionMessage De status van de boeking moet Concept zijn
+     */
+    public function testDeletionWorks()
```

Comment:
This sounds more like `testDeleteThrowsWhenResponseContainsErrorMessages`.

Can we test the happy path as well?

Reply:
I don't think it adds to much value, there's no special code path for happy path. 

If you insist, you can provide me with a successful response and I'll create a test around it.